tests/reg_tests/test_DVConstraints.py

```diff
@@ -113,6 +113,10 @@ def setUp(self):
         self.base_path = os.path.dirname(os.path.abspath(__file__))
 
     def generate_dvgeo_dvcon_rect(self, addToDVGeo=False):
+        """
+        The rectangular box is primarily used to test unscaled constraint function
+        values against known values for thickness, volume, and surface area.
+        """
```

Comment:
Nice! Are the comments on the rest of the functions just redundant, and so you opted to remove then? I think we could use a one-liner for all the function here, even if the names are pretty self explanatory

Reply:
Yes, I felt the comments were redundant. Some were also incorrect (saying that a constraint does not rely of point sets when it does). What did you have in mind for the one-liners? I think they would be almost exactly the same as the test name.

Reply:
yeah I see, don't worry, this is fine